traffic_ops/app/db/migrations/20170825124926_ignore_routing.sql

```diff
@@ -0,0 +1,14 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE profile
+    ADD ccr_routing_disabled BOOL,
```

Comment:
how about routing_disabled instead?

Reply:
maybe you should make it like this:

create a not null column called routing_disabled with default=false

then in the api if routingDisabled is not defined, set it's value to false for the insert/update...